platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/state/RandomSignedStateGenerator.java

```diff
@@ -463,7 +463,7 @@ public RandomSignedStateGenerator setPcesRound(final boolean pcesRound) {
 
     /**
      * Set if this state should use a {@link BlockingState} instead of a {@link MerkleStateRoot}.
-     * This flag is fasle by default.
+     * This flag is false by default.
```

Comment:
Actually a bit surprised I haven't seen this more in this PR - it's a common typo I've seen plenty of times in other codebases.